shared/chat/conversation-list/choose-conversation.native.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import type {Props} from './choose-conversation'
+
+export default (props: Props) => (
+  <Kb.Box2 direction="horizontal" centerChildren={true} style={{padding: 32}}>
+    <Kb.Text type="BodySmallError">
+      choose-conversation is not supported on mobile. If you see this outside storybook something is wrong.
```

Comment:
why even have this? just use choose-conversation.desktop (it works on both) and just never route here

Reply:
I was just thinking if someone makes a mistake and uses this on native, this'd make it very clear that it shouldn't happen that way.

Reply:
but then we need a .js.flow and a .native that does nothing. lets just not have it